.github/workflows/release.yml

```
@@ -9,54 +9,44 @@ on:
     - cron: '31 13 * * 3'
 
 jobs:
-  Build:
+  Version:
     outputs:
       created: ${{ steps.daily-version.outputs.created }}
+      version: ${{ steps.daily-version.outputs.version }}
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v2
         with:
           fetch-depth: 20
       - run: npm install
-      - run: npm test # This includes the build
+      - run: npm test
       - uses: fregante/daily-version-action@v1
         name: Create tag if necessary
         id: daily-version
       - uses: notlmn/release-with-changelog@v3
```

I think this should avoid this issue: https://github.com/notlmn/browser-extension-template/commit/15d944c8b051da517380043aff11903c880bddf4#commitcomment-43089079

```suggestion
      - uses: notlmn/release-with-changelog@v3
        if: steps.daily-version.outputs.created
```